spec/renderer/integration/store/TimelineSpace/Contents/Mentions.spec.ts

```diff
@@ -237,7 +244,7 @@ describe('Mentions', () => {
       }
     })
     it('should return only mentions', () => {
-      const mentions = store.getters['Mentions/mentions']
+      const mentions = store.getters['TimelineSpace/Contents/Mentions/mentions']
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/124234495/pulls/3280#issue-6896390))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/124234495/pulls/3280#issue-6896390) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.